### PR TITLE
Witch joke

### PR DESCRIPTION
**Pull request by alex-d-boyd**
Since it's coming up to Halloween, have a witch joke :)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -708,3 +708,8 @@ What's the best tool for automatically ignoring long email threads about tech bu
 
 ---
 
+**Q**: Why should you avoid the number 57005 around witches?
+
+**A**: In case one casts a hex on you.
```
Comment (styfle) [inline readme.md]: I thinks this could use some improvement. Are you trying to imply 57005 is 0xDEAD?

Comment (alex-d-boyd) [reply readme.md]: Yep, that's the starting point.  That and 'hex' having a second meaning as a malevolent spell.

I'll see if I can think of a way to improve it; suggestions welcome :)

Comment (styfle) [reply readme.md]: Q: What good can come of 2989 witches casting a hex?

A: None, it is always 0xBAD

You could also do something about the difference between 57005 and 57007 being DEAD vs DEAF

Comment (Joshua-rose) [reply readme.md]: Walking 57005?

**Pull request merged**